### DISCUSSION TITLE
Document limitations for big tilemaps

### DISCUSSION THREAD

**Discussion by Elogain** [StarArawn/bevy_ecs_tilemap#397] [Ideas]
I ran into an issue that it took me a while to figure out.

I have a single png file with tiles in it. It is 2048 * 1408 res, tilesize is 32*32, ending up with 2186 unique tiles, however, array_texture has a limitation of 2048. Perhaps have a section in the documentation mentioning this limitation (and possible other limitations I have not yet ran into).

**Comment by StarArawn** [comment]
Probably a better idea to have this as an issue instead of a discussion.
